holidays/countries/taiwan.py

```diff
@@ -103,3 +124,72 @@ class TW(Taiwan):
 
 class TWN(Taiwan):
     pass
+
+
+class TaiwanStaticHolidays:
+    # Date format (see strftime() Format Codes).
+    substituted_date_format = tr("%Y-%m-%d")
+    # Day off (substituted from %s).
+    substituted_label = tr("休息日（%s日起取代）")
```

Comment:
Is the trailing space intended here?

Reply:
I don't think that's a trailing space but [Fullwidth Right Parenthesis](https://www.compart.com/en/unicode/U+FF09) instead of the usual [Right Parenthesis](https://www.compart.com/en/unicode/U+0029) - something we should look into later for project-wide standardisation whether to keep or get rid of them.

Reply:
Update: I did look up the proper standard, and w3c sticks with fullwidth parenthesises for CJK languages

https://github.com/w3c/clreq/issues/74
https://japanese.stackexchange.com/questions/18855/when-mixing-japanese-and-english-should-i-use-japanese-parentheses

Reply:
Updated, now Fullwidth Parentheses are used everywhere for TW. Thank you for suggestion!

Reply:
All right, this seems to be resolved!